lib/rules/require-capitalized-constructors.js

```diff
@@ -32,16 +36,19 @@ var assert = require('assert');
 module.exports = function() {};
 
 module.exports.prototype = {
-
     configure: function(requireCapitalizedConstructors) {
         assert(
-            typeof requireCapitalizedConstructors === 'boolean',
-            'requireCapitalizedConstructors option requires boolean value'
-        );
-        assert(
-            requireCapitalizedConstructors === true,
-            'requireCapitalizedConstructors option requires true value or should be removed'
+            requireCapitalizedConstructors === true || Array.isArray(requireCapitalizedConstructors.allExcept),
+            'requireCapitalizedConstructors option requires object of exceptions or true value'
         );
+        this._allowedConstructors = {};
+
+        var allExcept = requireCapitalizedConstructors.allExcept;
```

Comment:
Well damn. Thanks for the tip :) I assumed booleans had a similar behavior to other primitives (e.g., `1.foobar` throws) – but ya know, JavaScript :)